tests/slow/src/test/scala/tests/sbt/SbtServerSuite.scala

```diff
@@ -0,0 +1,138 @@
+package tests.sbt
+
+import java.util.concurrent.TimeUnit
+
+import scala.meta.internal.builds.SbtBuildTool
+import scala.meta.internal.builds.SbtDigest
+import scala.meta.internal.metals.Messages
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.ServerCommands
+import scala.meta.internal.metals.{BuildInfo => V}
+import scala.meta.io.AbsolutePath
+
+import tests.BaseImportSuite
+import tests.ScriptsAssertions
+
+/**
+ * TODO make a BaseLspSuite powered by sbt
+ *
+ * Note that this entire suite will probably change. For now the entire way we
+ * have the Test server setup is to do a bloopInstall during initialize, but
+ * for these we don't really want that. For now since the flow will always be
+ * to favor Bloop to start, this is fine, and these tests all start with the
+ * assumption that we are starting with Bloop, and then switching. Eventually,
+ * We should probably have an entirely different BaseLspSuite powered by sbt.
+ * That will be a huge task in intself, so for now, I've decided to simply
+ * include these here and mark this as a TODO.
+ */
+class SbtServerSuite
+    extends BaseImportSuite("sbt-server")
+    with ScriptsAssertions {
+
+  val preBspVersion = "1.3.13"
+  // TODO after we go past 1.4.1 in Metals, this can just be pulled from
+  // V.sbtVersion
+  val bspVersion = "1.4.1"
```

Comment:
I will try to update to sbt 1.4.1. It's pretty annoying that we are blocked by docs.

Reply:
Awesome, thanks for taking care of this. Rebased on the latest changes and included this from our BuildInfo now.